deploy/craned/service.yaml

```diff
@@ -11,3 +11,31 @@ spec:
       targetPort: 9443
   selector:
     app: craned
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: crane-server-service
+  namespace: crane-system
+spec:
+  ports:
+    - port: 8082
+      protocol: TCP
+      targetPort: 8082
+  selector:
+    app: craned
+---
+apiVersion: v1
```

Comment:
I think we should only expose one service name crane. the service will expose multiple ports, include crane(which is the crane nginx web endpoint), craned-http (which is craned controller), craned web-http (which is craned web server), craned metrics-http (which is craned metrics)  @qmhu

Reply:
Ok